src/vehicle.h

```diff
@@ -891,6 +903,18 @@ class vehicle
      */
     float wheel_area( bool boat ) const;
 
+    /**
+     * Calculates the average radius of the wheels of the vehicle.
+     * @param boat If true, calculates the area under "wheels" that allow swimming.
+     */
+    float wheel_radius( bool boat ) const;
+
+    /**
+     * Calculates the mass of the wheels of the vehicle.
+     * @param boat If true, calculates the area under "wheels" that allow swimming.
```

Comment:
It calculates the *area* when `boat` is `true` and the *mass* when it's `false`?

Reply:
I forgot the edit the boat comments.